app/javascript/src/editable_components.js

```diff
@@ -801,8 +802,7 @@ function convertToMarkdown(html) {
  * For that reason, we are trying to be minimalistic in approach.
  **/
 function sanitiseHtml(html) {
-  html = html.replace(/<([\/\s])?script[^\<\>]*?>/mig, "&lt;$1script&gt;");
-  return html;
+  return sanitizeHtml(html);
 }
```

Comment:
I'm not wild about a function `sanitiseHtml()` calling a library `sanitizeHtml()` as its only one letter different.

However I wanted to leave our function intact and call the library from one place, as we are likely to add args to the sanitizeHtml library call.

resolve would be to rename our function, or to alias the library on require

Reply:
Hmm... yes, it's not ideal. My first thought was to suggest rename our function but we also have a sanitiseMarkdown(), so our code is being consistently with that. Perhaps go with the alias idea, then, or just live with it. I'll approve the PR and simply revisit if you change your mind.